x-pack/plugins/reporting/server/routes/internal/management/integration_tests/jobs.test.ts

```diff
@@ -175,7 +175,7 @@ describe(`Reporting Job Management Routes: Internal`, () => {
             ...licensingMock.createStart(),
             license$: new BehaviorSubject({ isActive: true, isAvailable: true, type: 'gold' }),
           },
-          security: { authc: { getCurrentUser: () => undefined } },
+          security: { authc: { getCurrentUser: () => undefined } }, // security comes from core here
```

Comment:
Let's use the `securityService` naming here, to distinguish it as the APIs that come from core.
```suggestion
          securityService: { authc: { getCurrentUser: () => undefined } },
```

Also, for the mocks on lines 101 and 392.